//This message is used for a request (or the associated response) for a virtual port to be opened or closed
message virtualPortRouterRequestOrResponse
{
optional uint32 virtualPortID = 1; //Which virtual port to open or close 
optional virtualPortOperationType operationType = 2; //This field tells the virtual port router to open or close the virtual port socket.  
optional bool operationSucceeded = 3; //This field is true if the operation succeeded and fails otherwise
optional string inprocVirtualPortAddress = 4; //The ZMQ inproc address the pair socket has been bound to
}

enum virtualPortOperationType
{
OPEN_VIRTUAL_PORT_SOCKET  = 1;
CLOSE_VIRTUAL_PORT_SOCKET = 2;
}
